lesson-000-web-server/README.md

```diff
@@ -45,3 +79,17 @@ And another one
 $ curl http://localhost:8080/hello/again/
 Greeting from alejandrox1-machine1!
 ```
+
+### Next steps (optional)
+
+As shown, we currently have one (root) route, which means: it works but not very practical.
```

Comment:
I think we should change the "not very practical."
It isn't a bad thing to have only a handler for `/`.